catroid/src/main/java/org/catrobat/catroid/web/ServerCalls.java

```diff
@@ -510,6 +513,24 @@ public void downloadProject(final String url, final String filePath, final Strin
 			httpClient.setConnectionSpecs(Arrays.asList(ConnectionSpec.CLEARTEXT));
 		}
 
+		if (receiver != null) {
```

Comment:
Yes, this happens intentionally when no progress updates in percent should be given to the user that is the case when download a template.